components/count-badge/test/count-badge-icon.vdiff.js

```diff
@@ -2,26 +2,97 @@ import '../count-badge-icon.js';
 import { expect, fixture, focusElem, html, oneEvent } from '@brightspace-ui/testing';
 
 describe('count-badge-icon', () => {
-	it('icon-skeleton', async() => {
-		const elem = await fixture(html`<d2l-count-badge-icon has-tooltip skeleton icon="tier3:gear" number="100"></d2l-count-badge-icon>`);
-		await expect(elem).to.be.golden();
+
+	describe('basic rendering', () => {
+		[
+			{ name: 'small-notification', template: html`<d2l-count-badge-icon icon="tier3:gear" size="small" type="notification" number="1" text="1 notification"></d2l-count-badge-icon>` },
+			{ name: 'large-notification', template: html`<d2l-count-badge-icon icon="tier3:gear" size="large" type="notification" number="5" text="5 notifications"></d2l-count-badge-icon>` },
+			{ name: 'small-count', template: html`<d2l-count-badge-icon icon="tier3:gear" size="small" type="count" number="42" text="42 items"></d2l-count-badge-icon>` },
+			{ name: 'large-count-icon', template: html`<d2l-count-badge-icon icon="tier3:gear" size="large" type="count" number="1" text="1 item"></d2l-count-badge-icon>` }
+		].forEach(({ name, template }) => {
+			it(name, async() => {
+				const elem = await fixture(template);
+				await expect(elem).to.be.golden();
+			});
+		});
+	});
+
+	describe('number truncation', () => {
+		[
+			{ name: 'small-notification-truncated-2-digits', template: html`<d2l-count-badge-icon icon="tier3:gear" size="small" type="notification" number="100" text="100 notifications"></d2l-count-badge-icon>` },
+			{ name: 'large-count-3-digits', template: html`<d2l-count-badge-icon icon="tier3:gear" size="large" type="count" number="123" text="123 items"></d2l-count-badge-icon>` },
+			{ name: 'large-count-4-digits', template: html`<d2l-count-badge-icon icon="tier3:gear" size="large" type="count" number="1234" text="1234 items"></d2l-count-badge-icon>` },
+			{ name: 'large-count-5-digits', template: html`<d2l-count-badge-icon icon="tier3:gear" size="large" type="count" number="12345" text="12345 items"></d2l-count-badge-icon>` },
+			{ name: 'large-number-centered', template: html`<d2l-count-badge-icon icon="tier3:gear" size="large" type="count" number="100000" text="100000 items"></d2l-count-badge-icon>` },
+			{ name: 'custom-max-digits-3', template: html`<d2l-count-badge-icon icon="tier3:gear" max-digits="3" number="5000" text="5000 items"></d2l-count-badge-icon>` },
+			{ name: 'custom-max-digits-4', template: html`<d2l-count-badge-icon icon="tier3:gear" max-digits="4" number="50000" text="50000 items"></d2l-count-badge-icon>` }
```

Comment:
I think we can likely get away with one one max-digits test but let me know if you have different thoughts